Source/OxyPlot/Axes/Axis.cs

```diff
@@ -543,6 +543,11 @@ public double Scale
         public bool UseSuperExponentialFormat { get; set; }
 
         /// <summary>
+        /// Gets or sets the natural dimensions of the axis, as of most recent rendering.
+        /// </summary>
+        public OxySize NaturalSize { get; protected set; }
```

Comment:
re: rebase, i tried that.  When I submitted this saw that included intermediate commits (sorry).  I do not know why the intermediate changes in the main repo show up here.  Obviously I did something wrong.  I followed the following guide:  

http://robots.thoughtbot.com/keeping-a-github-fork-updated